docs/logbook/LHC/2025_lhc.md

```diff
@@ -30,10 +30,11 @@ The tables below can be sorted by clicking next to the column headers.
 | 2025-04-21 17:30 | 2025-04-22 06:00 |    1.6HN     | Commissioning |                IRNL 1.2m and 18cm                |         [Shift Start](logbook://2025-04-21,1081,4240651) / [Summary](logbook://2025-04-22,1081,4240898) ([OP][op_irnl_1p2_18]{.cern_login target=_blank})          |
 | 2025-04-24 13:00 | 2025-04-25 03:30 |   0.6W 1WN   | Commissioning |   Global Corrs at 60cm and 18cm and Validation   |    [Shift Start](logbook://2025-04-24,1081,4243122) / [Summary](logbook://2025-04-25,1081,4243700) ([OP][op_global_corrs_60cm_18cm]{.cern_login target=_blank})    |
 | 2025-04-27 07:30 | 2025-04-27 23:30 |    1H 1HN    | Commissioning |             Validation of Everything             |    [Shift Start](logbook://2025-04-27,1081,4245465) / [Summary](logbook://2025-04-27,1081,4245531) ([OP][op_validation_everything]{.cern_login target=_blank})     |
-| 2025-05-07 20:00 | 2025-05-08 07:30 |    1.5WN     | Commissioning |                Ion Optics Part 1                 |       [Shift Start](logbook://2025-05-07,1081,4257152) / [Summary](logbook://2025-05-08,1081,4257277) ([OP][op_ion_optics_part1]{.cern_login target=_blank})       |
-| 2025-05-18 13:00 | 2025-05-18 19:00 | 0.5H 0.25HN  | Commissioning |                Ion Optics Part 2                 |       [Shift Start](logbook://2025-05-18,1081,4295338) / [Summary](logbook://2025-05-18,1081,4295555) ([OP][op_ion_optics_part2]{.cern_login target=_blank})       |
+| 2025-05-07 20:00 | 2025-05-08 07:30 |    1.5WN     | Commissioning |                Pb Ion Optics                     |       [Shift Start](logbook://2025-05-07,1081,4257152) / [Summary](logbook://2025-05-08,1081,4257277) ([OP][op_ion_optics_part1]{.cern_login target=_blank})       |
+| 2025-05-18 13:00 | 2025-05-18 19:00 | 0.5H 0.25HN  | Commissioning |                Pb Ion Optics                     |       [Shift Start](logbook://2025-05-18,1081,4295338) / [Summary](logbook://2025-05-18,1081,4295555) ([OP][op_ion_optics_part2]{.cern_login target=_blank})       |
```

Comment:
Would be nice to differentiate the names for these two, and propagate the new names at the end of file for the info dialogs.

I can do this if you're ok with me pushing to your branch.